staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -828,8 +838,12 @@ func (meal *cfgMeal) finishQueueSetReconfigsLocked() {
 		if limited := plState.pl.Spec.Limited; limited != nil {
 			if qCfg := limited.LimitResponse.Queuing; qCfg != nil {
 				meal.maxWaitingRequests += int(qCfg.Queues * qCfg.QueueLengthLimit)
+
+				handSize := float64(plState.pl.Spec.Limited.LimitResponse.Queuing.HandSize)
```

Comment:
Why define this variable that is only used once, instead of inlining its calculation?

Reply:
It was for readability cause the line was getting really long, but I guess i can just break the line instead :)